src/config/experiment.js

```diff
@@ -1,22 +1,45 @@
-import { defaultBlockSettings } from './main';
-import { deepCopy } from '../lib/utils'
+import { envConfig } from './main';
+import localConfig from "./config.json"
+import path from "path"
 
-// FIRST EXPERIMENT BLOCK SETTINGS
+const localConfigKeys = Object.keys(localConfig)
 
-// create copy of default settings
-let exptBlock1 = deepCopy(defaultBlockSettings)
-
-exptBlock1.repeats_per_condition = 2
-
-// SECOND EXPERIMENT BLOCK SETTINGS
-
-// create copy of default settings
-let exptBlock2 = deepCopy(defaultBlockSettings)
-
-exptBlock2.conditions = ["e", "f"]
-exptBlock2.repeats_per_condition = 2
+const checkConfig = (customConfig) => {
+  for (let i = 0; i < localConfigKeys.length; i++) {
```

Comment:
what is this checking? that all keys in the local config are in the custom config? couldn't we instead merge them so only a partial override was done with the participant specific config?